platform-includes/enriching-events/attach-screenshots/flutter.mdx

```diff
@@ -13,6 +13,8 @@ await SentryFlutter.init(
 );
 ```
 
+<PlatformContent includePath="replay/privacy-configuration" />
+
```

Comment:
would it work to move the privacy header out of the shared page and add some intro specific to screenshots? i.e. that masking is not enabled by default unless you specify the privacy options.

Reply:
I added a specific intro to Screenshots and Session Replay. I hope this is, what you had in mind.